core/server/src/main/java/alluxio/worker/netty/UnderFileSystemDataServerHandler.java

```diff
@@ -40,7 +40,7 @@
  * This class handles filesystem data server requests.
  */
 @NotThreadSafe
-public class UnderFileSystemDataServerHandler {
+class UnderFileSystemDataServerHandler {
```

Comment:
The other data server handlers are `final`. Is it possible for this to be final?